src/pages/Main/Main.tsx

```diff
@@ -3,10 +3,19 @@ import './Main.css';
 import { Route } from 'react-router-dom';
 import { GetStarted } from 'pages/GetStarted/GetStarted';
 import { AddPhoto } from 'pages/AddPhoto/AddPhoto';
+import { CanvasDiv } from './CanvasDiv';
 
 export const Main: React.FC= () => {
+
+    function renderCanvas() {
+        return <CanvasDiv ></CanvasDiv>
```

Comment:
Please remove the extra space ing <CanvasDiv >.

Reply:
done.